data/core/commands/doc.lua

```diff
@@ -36,9 +37,23 @@ local function save(filename)
     filename = core.normalize_to_project_dir(filename)
     abs_filename = core.project_absolute_path(filename)
   end
-  doc():save(filename, abs_filename)
-  local saved_filename = doc().filename
-  core.log("Saved \"%s\"", saved_filename)
+  local ok = pcall(doc().save, doc(), filename, abs_filename)
```

Comment:
I wonder if instead of a generic `pcall` we should actually check for the reason of the error. This way we could only handle actual save failures and other weird issues. Also would allow us to display the reason for the failure.

For now it's ok like this.

Reply:
maybe we could also just log the error at the same time, but i dont like how that looks though. either way as you said it's fine now and that can be done at a later time

Reply:
Actually yeah, log the error too.